### DISCUSSION TITLE
How to disable popup notifications on Fenix?

### DISCUSSION THREAD

**Discussion by JaredRichardWilliam** [uBlockOrigin/uAssets#22814] [Q&A]
I'm constantly notified that a redirect was blocked when navigating https://hianime.to/. I don't want to be notified, but also do not want to be redirected. https://superuser.com/a/1569676 appears to explain how to accomplish this, and the method works on my Windows 11 system using Chrome Stable. However, it doesn't work on my Android system using Firefox Nightly from the Play Store.

**Comment by gorhill** [comment]
uBO blocked a navigation triggered by either a webpage or another extension or some malware on your PC (see ["Does uBO open tabs on its own?"](https://github.com/gorhill/uBlock/wiki/Reference-answers-to-various-topics-seen-in-the-wild#does-ubo-open-tabs-on-its-own)). Disabling the "notification" means not blocking the navigation to what is probably a bad website, so this is not the solution, and the stackexchange answer just hides a symptom to a real issue.

I don't see any such unwanted navigation when I browse `https://hianime.to/` when using default settings/lists, and no other extension.

Please tell us what exactly you are doing which triggers the navigation to a blocked website, and share troubleshooting information as follow:

Go to the webpage with the issue > Click the [uBO toolbar icon](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface) > Click the [chat icon 💬](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > Click "Troubleshooting Information" > Report content here (https://www.reddit.com/link/173jmog/video/bogwf92af9tb1/player)

**Reply by roke-julian-lockhart** [reply]
> https://github.com/uBlockOrigin/uBlock-issues/discussions/3160#discussioncomment-8698214

@gorhill, this is his brother here. I originally filed this from his account so that he'd be notified.

Unfortunately, using that site can cause popups, but it's sporadic, for it's seemingly done by the server upon random mouse clicks. However, on any site which uses such an intrusive advertisement scheme, the behaviour is identical.

Can you elaborate on your first point – you appear to state that disabling pop-ups actually causes the user to visit those sites? It doesn't appear to, considering they close immediately upon redirection.

**Troubleshooting information**

<blockQuote>

```.log
uBlock Origin: 1.56.0
Firefox Mobile: 125
filterset (summary):
 network: 72378
 cosmetic: 37504
 scriptlet: 18449
 html: 1690
listset (total-discarded, last-updated):
 removed:
  ublock-privacy: null
  easyprivacy: null
  plowe-0: null
 default:
  user-filters: 0-0, never
  ublock-filters: 37775-363, 2h.13m Δ
  ublock-badware: 7754-14, 2h.13m Δ
  ublock-unbreak: 2302-1, 2h.13m Δ
  ublock-quick-fixes: 153-1, 2h.13m Δ
  adguard-mobile: 9113-50, 1h.13m
  easylist: 82039-16030, 2h.13m Δ
  urlhaus-1: 7405-5, 16h.16m
filterset (user): [empty]
trustedset:
 added: [array of 3 redacted]
userSettings:
 cloudStorageEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 2124 ms (selfie)
 maxAssetCacheWait: 566 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 0
```

</blockQuote>

**Reply by gorhill** [reply]
> disabling pop-ups actually causes the user to visit those sites?

No, it's just that the real cure is figuring how the site creates those popups and if possible create a filter which prevents it from creating them at all.

**Reply by gorhill** [reply]
In order to better investigate this filter issue -- we need to be able to reproduce in the first place -- it's best to use the chat icon in the popup panel to report it as a filter list issue in the proper repo, which is not here, filter issues are investigated/fixed at <https://github.com/uBlockOrigin/uAssets>.

So far I am unable to reproduce, and we already have filters for this site.

Also, I would suggest to **not** remove default list, we do not investigate filter issues when people remove default lists because we can't guarantee uBO functions as expected without these.

**Reply by garry-ut99** [reply]
- https://github.com/uBlockOrigin/uAssets/issues/22792
- https://github.com/AdguardTeam/AdguardFilters/issues/174237
- https://github.com/AdguardTeam/AdguardFilters/issues/173866

**Reply by roke-julian-lockhart** [reply]
> https://github.com/uBlockOrigin/uBlock-issues/discussions/3160#discussioncomment-8706864

@gorhill, I've only removed the privacy lists:

1.	<details>

	> ![image](https://github.com/uBlockOrigin/uBlock-issues/assets/42837531/6f119522-ca61-4b66-bafa-290847dd220f)

	*uBlock Filters - Privacy* and *Peter Lowe's*

1.	<details>

	> ![image](https://github.com/uBlockOrigin/uBlock-issues/assets/42837531/c90b642c-f4c0-4673-80fb-192faece4f2e)

1.	<details>

	> ![image](https://github.com/uBlockOrigin/uBlock-issues/assets/42837531/15d67d54-363d-43bb-917d-0c61dd4ad8c9)

	</details>

	*Sorry for the separate screenshots — see https://bugzilla.mozilla.org/show_bug.cgi?id=1884163#c0.*

...not least in order to prevent breakage (although also because I've little interest in privacy). Regardless, the issue persists even when they are enabled.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/uBlockOrigin/uBlock-issues/discussions/3160#discussioncomment-8698214

Regardless of whether the filters should be catching this, I would like to still know how to enable the pop-up prevention feature when using Fenix, @gorhill.

**Reply by krystian3w** [reply]
If page found method to avoid firewall `no-popups` and filter modifiers `random-word:faux_pass$popup,popunder` then you have to learn how to write scriptlets and analyze JS on websites.

You can still consider whether the phone is efficient enough for this site - This may reduce the race condition problem generated by oldie/weak CPU or RAM.
